### DISCUSSION TITLE
Cant install an app in site

### DISCUSSION THREAD

**Discussion by ufoloko** [2sic/2sxc#3062] [General] [answered]
I have exported an application from one site and im trying to install that application in another site, im receiving response as image below, but app is not appearing, resource files are not being created.

<img width="987" alt="Captura de pantalla 2023-03-31 a la(s) 17 43 54" src="https://user-images.githubusercontent.com/1165122/229267099-e3ad8640-e6cb-4a57-a9df-e214c6531b6b.png">

What can be happening

**Comment by iJungleboy** [comment]
Best check Insights to get an idea what's happening and where things get lost / or what's missing.

**Reply by ufoloko** [reply, marked as answer]
I noticed app was being exported in 2sxc version 14 and i was trying to install app in version 12. I had to recreate app exporting and importing each element from app.
